packages/logs/src/boot/startLogs.ts

```diff
@@ -164,7 +155,7 @@ export function doStartLogs(
         },
       }
     }
-    logger.log(log.message, messageContext, LogStatusForApi[log.api])
+    sender.sendHttpRequest(log.message, messageContext, LogStatusForApi[log.api])
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏I like that :)